e2e/smoke/test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash 
+
+# This file is used by https://github.com/bazel-contrib/.github/blob/master/.github/workflows/bazel.yaml
+
+# We don't check correctness of the lock generation here, but that the target exists and runs successfully.
+bazel run //multifile_config:lock_from_generated_config
```

Comment:
nit: I'm on the side of EOL on EOF
```suggestion
bazel run //multifile_config:lock_from_generated_config

```